test/Microsoft.OpenAPI.OData.Reader.Tests/Resources/Multiple.Schema.OpenApi.V2.yaml

```diff
@@ -411,8 +411,6 @@ paths:
         - Documents.Actions
       summary: Invoke action Upload
       operationId: Documents.DocumentDto.Upload
-      produces:
```

Comment:
this is probably a side effect of the serialization from openapi.net that'll need fixing is the produces property should still be here even though we only have referenced schemas. Can you coordinate things on that side please?

Reply:
It is. I'm working on a fix on the OpenAPI.NET library. Here's the [GitHub issue](https://github.com/microsoft/OpenAPI.NET/issues/1000) tracking this.